src/components/Pagination/Pagination.scss

```diff
@@ -16,3 +16,15 @@
     margin: 0 8px;
   }
 }
+
+.skip {
+  align-items: center;
+  display: flex;
+  justify-content: center;
+  
+  .button-skip {
+    color: #3d9dfe;
```

Comment:
PR looks great! One small change requested though...

instead of `color: #3d9dfe;`

you can do:

```
@include themeGen() {
  color: getThemeVal("textQuaternary");
}
```

The `themeGen` mixin is used to handle switching CSS rules from dark mode to light mode. If you look in the themes.scss file, you'll see that `textQuaternary` equals `$blueDarkText`, and if you look in colors.scss, you'll see `$blueDarkText` equals `#3d9dfe`.

We don't actually have light mode scss rules setup yet (you can see that around line 92 of the themes.scss file), but when we are ready to add light mode, all we need to do is add the light mode versions of all the colors that are listed above in the dark section of the themes.scss file.

Hope that makes sense! Let me know if you have questions.